diff --git a/src/utils.ts b/src/utils.ts
index 61357f41ee5a2f89ddf0f85151fd9ae6a9d2101d..2aa76722b754b3a125e418fa01209c11e5b02164 100644
--- a/src/utils.ts
+++ b/src/utils.ts
@@ -11,6 +11,7 @@ export async function _importRelative(installDir: string, packageWithPath: strin
 }
 
 export function _resolve(packageWithPath: string) {
+  const require = createRequire(import.meta.url)
   return require.resolve(packageWithPath)
 }
 